docs/api-overview.mdx

```diff
@@ -115,7 +122,125 @@ The same ApiKey necessary to call /hasMatchingCredentials grants you and only yo
 
 The `url` from the response ought to then be enriched with a `redirectUrl` query param that will take the next step in your target service's user onboarding webpage. _More information on this can be found in the [Acceptance Guide](/acceptance-guide)._
 
----
+:::info Example Credential Request
```

Comment:
I very much like this example section but we usually have left detailed examples for the guides... I suppose we may want to provide detailed example request bodies for all the api endpoints in this manner if we are doing for /hasMatchingCreds.

Reply:
Yeap, we can do this one day